internal/v1/api.yaml

```diff
@@ -573,6 +573,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        users:
+          type: array
+          items:
+            $ref: '#/components/schemas/User'
+    User:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: "user1"
+        key:
+          type: string
+          example: "userkey"
+        groups:
```

Comment:
Are there any gotchas with groups? What are the requirements for group support? Could the defaults be good enough so we don't need to expose this? I genuinely don't know, but if we look at how for instance amazon does user configuration I don't think it includes group customisation.

Reply:
Should the default be wheel? Is the default wheel? I feel I don't know as much as I should :)

Reply:
Right, I guess my question was more: should we allow selecting between sudo and non sudo users at all? Does it make sense to configure users without sudo access in this way?

Reply:
@gicmo thoughts on this?